cpp/src/copying/shift.cu

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018-2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ #include <cudf/copying.hpp>
+ #include <cudf/table/table.hpp>
+ #include <cudf/table/table_view.hpp>
+ #include <cudf/scalar/scalar.hpp>
+
+ namespace cudf {
+
+ std::unique_ptr<table> shift(const table_view& in,
```

Comment:
Just realized these should be `_ const&`, not `const _ &`. Still getting used to c++. :)

Reply:
East const is only a matter of preference. But I'm getting into the habit.

Reply:
I was wondering why I didn't see any issues pop up. Still reading my C++ books, and enjoying it thoroughly.